rmgpy/tools/sensitivity.py

```diff
@@ -73,8 +74,6 @@ def plotSensitivity(outputDirectory, reactionSystemIndex, sensitiveSpeciesList,
         ReactionSensitivityPlot(csvFile=csvFile, numReactions=number).barplot(reactionPlotFile)
         ThermoSensitivityPlot(csvFile=csvFile, numSpecies=number).barplot(thermoPlotFile)
 
-
```

Comment:
also here

Reply:
You mean make another commit and add the blank lines back in? Wouldn't that double clog the commit log?